service/storage_service.cc

```diff
@@ -410,8 +410,6 @@ future<> storage_service::topology_state_load() {
                     [[fallthrough]];
                 case topology::transition_state::commit_cdc_generation:
                     [[fallthrough]];
-                case topology::transition_state::publish_cdc_generation:
```

Comment:
`update_topology_state` does not handle `group0_concurrent_modification`. It only logs and re-throws.

Reply:
Right, I somehow missed this `throw;`. I fixed it.